README.md

```diff
@@ -11,57 +11,27 @@ Team A:
 
 ## Table of Contents
 
-- [General Information](#general-information)
-- [Project Timeline](#project-timeline)
-- [Tech Stack](#tech-stack)
-- [Setup](#setup)
-- [How it Works](#how-it-works)
-- [Limitations](#limitations)
+- [The Speaking Portal Project](#the-speaking-portal-project)
+  - [Table of Contents](#table-of-contents)
+  - [General Information](#general-information)
+  - [Tech Stack](#tech-stack)
+  - [Setup](#setup)
+  - [How it Works](#how-it-works)
+    - [Overview](#overview)
+    - [API](#api)
+    - [The Phoneme Factory](#the-phoneme-factory)
+      - [Rhubarb Output](#rhubarb-output)
+  - [The Animation Factory](#the-animation-factory)
+    - [Frame Data Output](#frame-data-output)
+  - [Limitations](#limitations)
 
 ## General Information
 
-The objective of the Speaking Portal Project is to design, develop, and deploy a lip-sync animation API for the
+The objective of the Speaking Portal Project is to design, develop, and deploy an API for the
 [Kukarella text-to-speech (TTS) web application](https://www.kukarella.com/). This API will serve as an
-animation-generating add-on for this system so that the user can both listen to and watch their avatar speak the user
+animation-generating add-on for this system so that the user can both listen to and watch an avatar speak the
```

Comment:
```suggestion
animation-generating add-on for this system. The generated animation will allow a user to watch an avatar speak the
```
Sentence is pretty wordy so would be nice to split, but not 100% on this alternative. Technically it's implied if you're watching it speak the provided text that you can listen to it I think?